.github/workflows/python-env-upload.yml

```diff
@@ -0,0 +1,37 @@
+# This workflows will upload a python environment container to quay when there are changes to master
+# For more information see: https://github.com/docker/build-push-action
+
+name: Upload Python Environment Container to Quay
+
+on:
+  push:
+    branches: master
+
+jobs:
+  multi:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: login to quay
+        run: echo "${{ secrets.QUAY_TOKEN }}" | docker login quay.io -u "${{ secrets.QUAY_USER }}" --password-stdin
+      - 
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          platforms: linux/amd64, linux/ppc64le
+          push: true
+          tags: |
+            quay.io/multicloudops/operator-csv-libs:latest
+            quay.io/multicloudops/operator-csv-libs:1.0.0
```

Comment:
This would always tag every build with `1.0.0` which wouldn't offer anything above what `latest` offers (i.e. folks wouldn't be guaranteed that working with 1.0.0 would be functionally the same tomorrow as it is today). I can see there's some discussion on the `tag_with_ref` feature that was in v1 of the action, and apparently has gone away, so I guess we could do something like this: https://github.com/marketplace/actions/generate-smart-tag-for-docker-images